src/styles/sidebar/components/search-input.scss

```diff
@@ -0,0 +1,46 @@
+.search-input__form {
+  display: flex;
+  flex-flow: row nowrap;
+
+  position: relative;
+  color: $gray-dark;
+}
+
+.search-input__icon {
+  order: 0;
+}
+
+.search-input__input {
+  @include outline-on-keyboard-focus;
+
+  flex-grow: 1;
+  order: 1;
+
+  color: $text-color;
+
+  // Disable default browser styling for the input.
+  border: none;
+  padding: 0px;
+  width: 100%;
+
+  // The search box expands when focused, via a change in the
+  // `max-width` property. In Safari, the <input> will not accept
+  // focus if `max-width` is set to 0px so we set it to
+  // a near-zero positive value instead.
+  // See GH #2654
```

Comment:
Which repo? `client`? Link instead, maybe?

Reply:
Ah, h. It predates the separation of h and the client into separate repos: https://github.com/hypothesis/h/issues/2654